spec/models/course_spec.rb

```diff
@@ -318,13 +318,13 @@
         it "Requires a level" do
           error = errors[:level]
           expect(error).not_to be_empty
-          expect(error.first).to include("You need to pick a level")
+          expect(error.first).to include("Select a level")
```

Comment:
Change to 'Select a course level' (if this relates to Primary/Secondary/Further education).